### ISSUE TITLE
Add land cover lookup

### ISSUE THREAD

**Issue by oubiwann** [hexagram30/map#28]
Note: some or all of this ticket is being superseded by https://github.com/hexagram30/raster/issues/15 ...

Once the biome lookup work is done (#23), let's add land cover support:
* Create a mapping of biomes to % land cover for types of vegetation. 
* Then create a map ... since each pixel covers a large square area (taking most of a day's travel to walk across), mix up the coverage. (E.g., If a biome has 10%, 15%, 50%, 5%, and 20% composition of different types of vegetation, make sure that each of those is somewhere between 50-80% the primary vegetation type, and comprised of some portion of the others. Since color will only indicate the primary vegetation cover, the secondary types will need to be tracked in a data store.)
* Add support for seasonal variation: season count, season types, percent coverage change during those times, maybe even completely different coverage in different seasons ...

**Comment by oubiwann** [comment]
#23 is done!!! Let the land cover work begin :-)

**Comment by oubiwann** [comment]
I want to pause on this until #33 is resolved.

**Comment by oubiwann** [comment]
Actually, #33 has been closed and won't be done. Per #30, a new service is being created to support raster layers. The map project will utilize the following:
* The raster service's networked storage (see https://github.com/hexagram30/raster/issues/3)
* The raster gRPC service (see https://github.com/hexagram30/raster/issues/11)
* a Clojure library for the gRPC service (see https://github.com/hexagram30/raster/issues/8)

As such, a companion ticket has been created here:
* https://github.com/hexagram30/raster/issues/15

I will change the description of the current ticket to reflect this ...

**Comment by oubiwann** [comment]
Closing in favour of https://github.com/hexagram30/raster/issues/21
